src/components/App/index.tsx

```diff
@@ -0,0 +1,202 @@
+import React, { useContext } from 'react'
+import { withStyles } from '@material-ui/core/styles'
+import { SnackbarProvider } from 'notistack'
+import { connect, useSelector } from 'react-redux'
+import { useRouteMatch, useHistory } from 'react-router-dom'
+
+import AlertIcon from './assets/alert.svg'
+import CheckIcon from './assets/check.svg'
+import ErrorIcon from './assets/error.svg'
+import InfoIcon from './assets/info.svg'
+import styles from './index.module.scss'
+
+import SidebarLayout from 'src/components/SidebarLayout'
+import Header from 'src/components/SidebarLayout/Header'
+import Sidebar from 'src/components/SidebarLayout/Sidebar'
+import { SidebarContext } from 'src/components/Sidebar'
+import CookiesBanner from 'src/components/CookiesBanner'
+import Notifier from 'src/components/Notifier'
+import SidebarProvider from 'src/components/Sidebar'
```

Comment:
Same thought here with Sidebar. Are we planning to use Sidebar outside SidebarLayout?

Reply:
same as above. I think renaming it will help to understand.

Reply:
done